edx_proctoring/templates/proctoring/seq_timed_exam_expired.html

```diff
@@ -0,0 +1,31 @@
+{% load i18n %}
+<div class="critical-time sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You did not complete the exam in the allotted time
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since you did not submit your exam before the time allotted expired, your work cannot be
+      considered as an exam that is eligible for credit. Any work you've completed may still be counted for
+      partial credit towards your grade.
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <p>
+      {% blocktrans %}
+        Please see <a href="{{progress_page_url}}">your progress in this course</a> for your general course credit worthiness.
+      {% endblocktrans %}
+    </p>
+  </div>
+</div>
+<div class="footer-sequence border-b-0 padding-b-0">
+  <span> {% trans "Is there anything i can do to make up/replace this session ?" %} </span>
```

Comment:
Thanks, @muhhshoaib. I think these "timed only" templates are not going to be visible to course teams or students in this initial release so I will hold off on edits for now.